packages/api/src/command/medical/document/process-doc-query-webhook.ts

```diff
@@ -54,55 +100,30 @@ export const processDocQueryProgressWebhook = async ({
   }
 };
 
-const handleDownloadWebhook = async (
-  webhookSent: boolean,
-  patient: Pick<Patient, "id" | "cxId" | "externalId">,
-  requestId: string,
-  documentQueryProgress: DocumentQueryProgress
-): Promise<void> => {
-  const downloadStatus = documentQueryProgress.download?.status;
-  const isDownloadFinished = downloadStatus === "completed" || downloadStatus === "failed";
-
-  const canProcessRequest = isDownloadFinished && !webhookSent;
-
-  if (canProcessRequest && !isSandbox) {
-    const downloadIsCompleted = downloadStatus === "completed";
-    const payload = await composeDocRefPayload(patient.id, patient.cxId, requestId);
-
-    processPatientDocumentRequest(
-      patient.cxId,
-      patient.id,
-      DOWNLOAD_WEBHOOK_TYPE,
-      downloadIsCompleted ? MAPIWebhookStatus.completed : MAPIWebhookStatus.failed,
-      requestId,
-      downloadIsCompleted ? payload : undefined
-    );
-  }
-};
-
-const handleConversionWebhook = async (
-  webhookSent: boolean,
-  patient: Pick<Patient, "id" | "cxId" | "externalId">,
-  requestId: string,
-  documentQueryProgress: DocumentQueryProgress
-): Promise<void> => {
-  const convertStatus = documentQueryProgress.convert?.status;
-  const isConvertFinished = convertStatus === "completed" || convertStatus === "failed";
-
-  const canProcessRequest = isConvertFinished && !webhookSent;
-
-  if (canProcessRequest) {
-    const convertIsCompleted = convertStatus === "completed";
+async function handleWebhook({
```

Comment:
I feel like this function doesnt really add anything and is an unnessecary abstraction around `processPatientDocumentRequest` that makes following the code a little bit harder

Reply:
Like all places that call it could just call `processPatientDocumentRequest` directly

Reply:
Fair point will update